src/main/java/com/azure/cosmos/examples/common/UserSession.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.common;
+
+public class UserSession {
+    public UserSession()
```

Comment:
If there is no other constructor specified in this class, you don't need to add an empty constructor. Java will add one by default.